### DISCUSSION TITLE
Removal of examine option

### DISCUSSION THREAD

**Discussion by Pvff13** [runelite/runelite#15142] [Q&A] [answered]
I think having an option to remove the examine option would be nice, when right clicking a lot of npcs/objects it clutters up your screen with a larger menu and a lot of the time players only use it to price check stuff

**Comment by Nightfirecat** [comment, marked as answer]
The [No Examine hub plugin](https://runelite.net/plugin-hub/show/no-examine) was created just a few days ago, and does exactly that.
